docs/flask_config.md

```diff
@@ -76,3 +76,50 @@ description sources not mentioned in the configuration will be alphabetically pl
 
 Here is a screenshot of what it would look like in the bottom left here:
 ![programmatic_description](img/programmatic_descriptions.png)
+
+##Uneditable Table Descriptions
+Amundsen supports configuring table and column description to be non-editable for selective tables. You may want to make table 
+descriptions non-editable due to various reasons such as table already has table description from source of truth.
+You can define matching rules in `config file` for selecting tables. This configuration is useful as table selection criteria can 
+be company specific which will not directly integrated with Amundsen. 
+You can use different combinations of schema and table name for selecting tables.
+
+Here are some examples when this feature can be used: 
+1. You want to set all tables with a given schema pattern as un-editable.
```

Comment:
nit: `You want to set all tables with a given schema  or schema pattern as un-editable.`